cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/s3_dataset.spec.js

```diff
@@ -141,6 +147,40 @@ const s3DatasetTestSuite = () => {
           cy.getElementByTestId('docTable').should('be.visible');
           cy.getElementByTestId('docTable').find('tr').should('have.length', 11);
         });
+
+        it('aborts and cancels previous query when new query is started', function () {
+          cy.getElementByTestId(`datasetSelectorButton`).click();
+          cy.getElementByTestId(`datasetSelectorAdvancedButton`).click();
+
+          cy.get(`[title="S3 Connections"]`).click();
+          cy.get(`[title="BasicS3Connection"]`).click();
+          cy.get(`[title="mys3"]`).click();
+          cy.get(`[title="default"]`).click();
+          cy.get(`[title="http_logs"]`).click();
+          cy.getElementByTestId('datasetSelectorNext').click();
+          cy.get(`[class="euiModalHeader__title"]`).should('contain', 'Step 2: Configure data');
```

Comment:
nit: given that this test just verify if it aborts  verifyhing if selecting the title has this increases potential points of failure unrelated to this test.

in this test it would be better to just navigate to the page with a s3 query run it and then run it again focusing on the test name.

but this should be addressed later not now.